app/views/agent/ReturnToClientDetails.scala.html

```diff
@@ -0,0 +1,59 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.AppConfig
+@import views.html.helpers.injected.{ContinueButton, RadioHelper}
+@import views.html.templates.AgentMainTemplate
+@import forms.agent.ReturnToClientDetailsForm
+@import models.ReturnToClientDetailsModel._
+
+@this(
+    mainTemplate: AgentMainTemplate,
+    radioHelper: RadioHelper,
+    continueButton: ContinueButton,
+    form : FormWithCSRF
+)
+
+@(returnToClientDetailsViewForm: Form[ReturnToClientDetailsModel], postAction: Call, clientName: String)(implicit request: Request[_], messages: Messages)
+
+
+@mainTemplate(title = messages("agent.return-to-client-details.title"), optForm = Some(returnToClientDetailsViewForm)) {
+  @form(action = postAction) {
+
+    <h1 class="govuk-heading-l">@messages("agent.return-to-client-details.heading")</h1>
+
+
+    @radioHelper(
+      field = returnToClientDetailsViewForm(ReturnToClientDetailsForm.returnToClientDetails),
+      legend = messages("agent.return-to-client-details.heading"),
+      isPageHeading = false,
+      headingClasses = "govuk-visually-hidden",
+      radioItems = Seq(
+        RadioItem(
+            content = Text(ContinueWithCurrentClient.toMessageString(clientName)),
+            value = Some(ContinueWithCurrentClient.toString)
+        ),
+        RadioItem(
+            content = Text(SignUpAnotherClient.toMessageString()),
+            value = Some(SignUpAnotherClient.toString)
+        )
+      )
+    )
```

Comment:
```suggestion
    @radioHelper(
      field = returnToClientDetailsViewForm(ReturnToClientDetailsForm.returnToClientDetails),
      legend = messages("agent.return-to-client-details.heading"),
      isPageHeading = true,
      radioItems = Seq(
        RadioItem(
            content = Text(ContinueWithCurrentClient.toMessageString(clientName)),
            value = Some(ContinueWithCurrentClient.toString)
        ),
        RadioItem(
            content = Text(SignUpAnotherClient.toMessageString()),
            value = Some(SignUpAnotherClient.toString)
        )
      )
    )
```
We can make the legend of the radio input the heading for the page with the above change.